modules/installation/pages/network-requirements.adoc

```diff
@@ -1,31 +1,26 @@
 [[installation-network-requirements]]
 = Network Requirements
 
-
-
-
-
-
 This section details the networking and port requirements for {productname}.
 
 Fully Qualified Domain Name (FQDN)::
 The {productname} server must resolve its FQDN correctly or cookies will not work properly on the WebUI.
 +
-For more information about configuring the hostname and DNS, see https://www.suse.com/documentation/sles-12/book_sle_admin/data/sec_basicnet_yast.html#sec_basicnet_yast_change_host[SUSE Linux Enterprise Server Documentation - Configuring Host Name and DNS]
+For more information about configuring the hostname and DNS, see link:https://www.suse.com/documentation/sles-15/book_sle_admin/data/sec_network_yast.html#sec_network_yast_change_host[SUSE Linux Enterprise Server Documentation - Configuring Host Name and DNS]
 
 Hostname and IP Address::
 To ensure that the {productname} domain name can be resolved by its clients, both server and client machines must be connected to a working DNS server.
 +
-For more information about setting up a DNS server, see https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_dns.html[SUSE Linux Enterprise Server Documentation - The Domain Name System]
+For more information about setting up a DNS server, see link:https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_dns.html[SUSE Linux Enterprise Server Documentation - The Domain Name System]
 
 Using a Proxy When Installing from {sle} Media::
 If you are on an internal network and do not have access to {scc}, you can set up and use a proxy during installation.
 +
-For more information about configuring a proxy for access to {scc} during a {sle} installation, see https://www.suse.com/documentation/sled-12/singlehtml/book_sle_deployment/book_sle_deployment.html#sec.i.yast2.start.parameters.proxy[SUSE Linux Enterprise Server Documentation -  Using a Proxy During Installation]
+For more information about configuring a proxy for access to {scc} during a {sle} installation, see link:https://www.suse.com/documentation/sles-15/book_sle_deployment/data/sec_boot_parameters_advanced.html#sec_boot_parameters_advanced_proxy[SUSE Linux Enterprise Server Documentation -  Using a Proxy During Installation]
 
 
 .Naming Your Server
-[IMPORTANT]
+[CAUTION]
```

Comment:
```suggestion
[IMPORTANT]
```

We have Note, Important, and Warning.

Reply:
We have all five admon types? 

Note, Tip, Important, Warning and Caution.
Caution is something that can introduce breaking changes?

Reply:
I agree that caution here was incorrect. However we probably need to define our use of admons in our styleguide.